Configure.pl

```diff
@@ -361,7 +368,10 @@ ($$)
     ? '-Rpass=loop-vectorize'
 : $config{cc} eq 'gcc'
     ? '-fopt-info-vec-optimized'
-    : die if $args{'show-autovect'};
+  : die if $args{'show-autovect'};
+# TODO - we need to do -fno-omit-frame-pointer anyway if we run the JIT
```

Comment:
If we're not stack walking, is this still the case?

Reply:
No, this isn't the case anymore.